src/content/en/2024/fonts.md

```diff
@@ -457,9 +457,9 @@ Emoji fonts do not suffer from this problem as emoji are encoded in Unicode so t
 
 ## Families and foundries
 
-Which font families were most popular this year, and which foundries made them? Taking a look at the top 20 there are few surprises in the first ten entries compared to 2022. Roboto still leads the pack, with a small increase in use\*. Font Awesome use has remained fairly stable, and so has Noto usage (unlike 2022, Noto is now split into script specific items). The only surprise is the decline of Lato, which has been overtaken by Poppins and Montserrat.
+Which font families were most popular this year, and which foundries made them? Taking a look at the top 20 there are few surprises in the first ten entries compared to 2022. Roboto still leads the pack, with a small increase in use*. Font Awesome use has remained fairly stable, and so has Noto usage (unlike 2022, Noto is now split into script specific items). The only surprise is the decline of Lato, which has been overtaken by Poppins and Montserrat.
 
-<p class="note">\* As noted in 2022 the discrepancy between desktop and mobile use of Roboto is mostly likely due to the use of `local()` which loads a locally installed version of Roboto; due to it being the system font on Android, usage on mobile is low.</p>
```

Comment:
@tunetheweb This one is actually necessary, otherwise it turns into a bullet point.

Reply:
Oops. Thought I'd tested that. Fixed in #3849